algorithms/unit_tests/CMakeLists.txt

```diff
@@ -33,57 +33,42 @@ IF(CMAKE_CXX_CLANG_TIDY)
   SET_TARGET_PROPERTIES(kokkosalgorithms_gtest PROPERTIES CXX_CLANG_TIDY "")
 ENDIF()
 
-SET(SOURCES
-  UnitTestMain.cpp
-)
+SET(ALGORITHM UnitTestMain.cpp)
 
 IF(Kokkos_ENABLE_OPENMP)
-  LIST( APPEND SOURCES
-    TestOpenMP.cpp
+  LIST(APPEND ALGORITHM_SOURCES
     TestOpenMP_Sort1D.cpp
     TestOpenMP_Sort3D.cpp
     TestOpenMP_SortDynamicView.cpp
```

Comment:
`TestOpenMP.cpp` is merged into `TestOpenMP_Sort1D.cpp` and `TestOpenMP_Randonm.cpp` is the test corresponding to `TestBACKEND.cpp` for the other backends.